doc/efa-env-var.md

```diff
@@ -0,0 +1,130 @@
+# EFA Cheatsheet
+
+## 1. Settings via environment variables
+
+For optimized performance, you may need to set additional environment variables depending on the
+versions of your libfabric.
+
+<table>
+   <thead>
+      <th>Setting</th>
+      <th>Description</th>
+   </thead>
+   <tr>
+      <td><code>FI_EFA_USE_HUGE_PAGE=0</code></td>
+      <td>Set to 0 when you see <code>os.fork()</code> causes <code>OSError: Cannot allocate
```

Comment:
Formatting nit: newlines and spaces are preserved in `<code>` blocks, so we shouldn't wrap this line inside the code block (See GitHub's markdown preview for this file.)